cfgov/unprocessed/js/routes/on-demand/simple-chart/simple-chart.js

```diff
@@ -61,6 +61,7 @@ function fetchData( url, isCSV ) {
 
     return prom.then( d => {
       if ( isCSV ) {
+        d = d.replace( /\n"(#|\/\/)/g, '$1' );
```

Comment:
This won't remove a leading quotation mark on the first line of the file, since the first line doesn't have `\n` at the start.

Reply:
Good catch!